resources/views/dues/payDues.blade.php

```diff
@@ -0,0 +1,14 @@
+@extends('layouts/app')
+
+@section('title')
+Dues | {{ env('APP_NAME') }}
```

Comment:
In case we do ever end up doing config caching, this should get changed to use the `config()`  accessor for values rather than `env()` directly.